core/cli/src/lib.rs

```diff
@@ -434,7 +434,7 @@ where
 		return Ok(Action::ExecutedInternally);
 	} else if let Some(sub_matches) = matches.subcommand_matches("revert") {
 		revert_chain::<F>(
-			get_db_path_for_subcommand(matches, sub_matches, app_info)?,
+			&config.database_path,
```

Comment:
All sub-commands use this function, so could update all of them to use this fix? And remove `get_db_path_for_subcommand`.

Reply:
Done.